HW04.Rmd

```diff
@@ -69,51 +152,123 @@ Assume that:
 * Fall = October-December
 
 ```{r season}
+gun_deaths_season <- gun_deaths %>% 
+    select(month) %>% 
+    count(month)
+    
+gun_deaths_season$month <- month_abbrev
 
+gun_deaths_season2 <- gun_deaths_season %>% 
+    summarize(Winter = sum(n[1:3]), Spring = sum(n[4:6]), Summer = sum(n[7:9]), Fall = sum(n[10:12]))
+
+kable(gun_deaths_season2, format = "markdown")
 ```
 
+This table shows that Summer has the most total gun deaths from 2012-2014
 
 # These are more open ended questions from here on out, you could look at making multiple summaries or types of plots. 
 
 ### Are whites who are killed by guns more likely to die because of suicide or homicide? How does this compare to blacks and hispanics?
 
 ```{r race}
-
+gun_death_s_h <- gun_deaths %>% 
+    select(race, intent) %>% 
+    filter(race == c("White", "Black", "Hispanic"), intent == c("Suicide", "Homicide")) %>% 
+    group_by(race) %>% 
+    count(intent) %>% 
+    mutate(total_death = sum(n)) %>% 
+    mutate(fraction_intent = n / total_death)
+
+gun_death_s_h %>% 
+    ggplot(aes(x = race, y = fraction_intent, fill = intent)) +
+    geom_col() +
+    labs(title = "proportion of Gun Deaths that are Homicides or Suicides by Race, 2012-2014",
+         x = "Race",
+         y = "Proportion") +
+    scale_y_continuous(expand = c(0,0))

```

Comment:
I wasn't sure what the expand argument did, but now I see that it eliminated that blank space under the bars; that's super useful to know!